test/UnitTests/MSTestAdapter.UnitTests/Execution/TestClassInfoTests.cs

```diff
@@ -491,7 +491,7 @@ public void RunClassCleanupShouldReturnAssertFailureExceptionDetails()
         _testClassInfo.ClassCleanupMethod = typeof(DummyTestClass).GetMethod(nameof(DummyTestClass.ClassCleanupMethod));
 
         // Act
-        var classCleanup = _testClassInfo.RunClassCleanup();
+        var classCleanup = _testClassInfo.RunClassCleanup(UTF.ClassCleanupBehavior.EndOfAssembly);
```

Comment:
Does it need to be changed? If it does, are we having some equivalent test for the other beahviour?

Same comment for the other tests.

Reply:
yes because it was initially testing the `EndOfAssembly` behavior and thrown exception is in it's flow not in the `endofclass's` flow. 

and yeah most of the remaining tests now tests the `endofclass` behavior as it's the default.